### DISCUSSION TITLE
I Need A Clone Page Of Storageapi.fleek.co

### DISCUSSION THREAD

**Discussion by Enas991** [community/community#39191] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I just opened a new fleek account and I need exactly the one on storageapi.fleek.co. I need help with the process

**Comment by david-engelmann** [comment]
[Fleek.co has pretty great docs that can be found here](https://docs.fleek.co/). This discussion section is mainly for Github specific issues, posting your question in a Fleek.co specific github repository will yield better results. [You can pick the relevant repository for your question from here](https://github.com/FleekHQ)!

**Comment by Enas991** [comment]
Can you help with the clone script?

**Reply by david-engelmann** [reply]
absolutely, post the details

**Comment by Enas991** [comment]
I want to create a similar website as storageapi.fleek.co.. I have a
fleek.co account but don't know how to do it
